src/documentation/index.md

```diff
@@ -46,6 +46,7 @@ A Beam Runner runs a Beam pipeline on a specific (often distributed) data proces
 * [SparkRunner]({{ site.baseurl }}/documentation/runners/spark/): Runs on [Apache Spark](http://spark.apache.org).
 * [DataflowRunner]({{ site.baseurl }}/documentation/runners/dataflow/): Runs on [Google Cloud Dataflow](https://cloud.google.com/dataflow), a fully managed service within [Google Cloud Platform](https://cloud.google.com/).
 * [GearpumpRunner]({{ site.baseurl }}/documentation/runners/gearpump/): Runs on [Apache Gearpump (incubating)](http://gearpump.apache.org).
+* [SamzaRunner]({{ site.baseurl }}/documentation/runners/samza/): Runs on [Apache Samza](http://samza.apache.org).
```

Comment:
ure. Since Samza runner is already on master, I assume it will go out as 2.6.0 release? So we will update this part after the release, if I understand correctly.

Reply:
It is OK to have things on the site that are not yet released. We haven't really got a good process for that. So we put stuff up when it hits master, usually, and sometimes with a little note. You can leave it here. As a follow-up, they should be in alphabetical order.